ui/src/message_popup/hint_card.jsx

```diff
@@ -43,35 +44,39 @@ export default React.createClass({
     const {hidden, allExamples} = this.state;
     return (
       <div>
-        {hidden &&
-          (<div>
-            <div style={styles.buttonRow}>
-              <div></div>
-              <RaisedButton
-                onTouchTap={this.onHintsToggled}
-                style={styles.button}
-                secondary={true}
-                label="Show Example" />
-            </div>
-          </div>)
-        }
-        {!hidden &&
-          (<div>
-            <div style={styles.exampleBox}>
+        <VelocityTransitionGroup leave={{animation: "slideUp"}} runOnMount={true}>
```

Comment:
This one felt a little clunky and seems like it might be more natural for it to `slideDown` from where the user clicks the button.  And am I following that the idea of the two different transition groups here is that the first time there's an animation but the second time there isn't, or it's different?  I'm wondering if it could be a single transition group with examples that `slideDown` when they enter and exit, would that make sense?